### ISSUE TITLE
Introduce  bocadillo

### ISSUE THREAD

**Issue by waghanza** [the-benchmarker/web-frameworks#951]
+ Language : python
+ Repo : https://github.com/bocadilloproject/bocadillo
+ Author :  @florimondmanca

**Comment by florimondmanca** [comment]
Neat idea! Would love to see how Bocadillo compares to other frameworks. ;-)

Food for thought and implementation ideas: https://github.com/bocadilloproject/benchmarks

**Comment by waghanza** [comment]
@florimondmanca With the last `starlette` release (**0.11**), I have a 
```
ModuleNotFoundError: No module named 'starlette.middleware.lifespan'
```
using **0.10.6** the code here is OK

**Comment by florimondmanca** [comment]
Hey @waghanza thanks for reporting. I just spotted that an hour ago too and just released a hot fix in v0.11.1. You can upgrade to v0.11.1 and things should go back to normal!

**Comment by waghanza** [comment]
I'll leave that to @dependabot[bot] :stuck_out_tongue: (it will create a `PR` for that)
